server/models/db/db.ts

```diff
@@ -2,6 +2,6 @@
 import 'dotenv/config';
 import { drizzle } from 'drizzle-orm/connect';
 
-const db = drizzle('libsql', process.env.DB_FILE_NAME as string);
+const db = await drizzle('libsql', { connection: process.env.DATABASE_URL, casing: 'snake_case' });
```

Comment:
Do we need the await here? Or is it enough to import (non-awaited) db and use await in the controllers

Reply:
not totally sure - do you have a preference/do you know if one is better than the other?